deploy_flask.sh

```diff
@@ -10,9 +10,8 @@ pip3 install Flask
 
 apt install -y postgresql postgresql-contrib
 
-cd /vagrant/
-
-FLASK_APP=hello.py
-python3 hello.py >> log.txt 2>&1 &
+FLASK_APP=/vagrant/hello.py
+python3 hello.py >> /vagrant/log.txt 2>&1 &
 
 exit 0
+>>>>>>> master
```

Comment:
Have you tested the application (spinned up a DEV environment) after the rebase?
This line might causing you issues.

Reply:
Testing and uploading new commit